### PR TITLE
[MIG] 11.0 mig partner company type

### PR DESCRIPTION
**Pull request by daramousk**

### CODE REVIEW

File: partner_company_type/__manifest__.py
Comment (nikul-serpentcs) [inline partner_company_type/__manifest__.py]: @daramousk `website` should be [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

File: partner_contact_birthdate/models/__init__.py
Comment (nikul-serpentcs) [inline partner_contact_birthdate/models/__init__.py]: @daramousk remove `Copyright` from all `__init__` file

File: partner_contact_birthdate/models/res_partner.py
Comment (nikul-serpentcs) [inline partner_contact_birthdate/models/res_partner.py]: IMO here change `birthdate_date`  to `birthdate`. For more worth reading.

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

**Comment by daramousk** [comment]
@hbrunn The tests do not actually fail but they are marked as a failure. See assertRaises. Also, I am not sure what to do with the CLA not being signed.

**Comment by hbrunn** [comment]
wrap it in https://github.com/OCA/OCB/blob/11.0/odoo/tools/misc.py#L783

The mail address is @Yajo, so no problem. Do we have a possibility to declare aliases of people for the CLA bot?

**Comment by pedrobaeza** [comment]
Uhm, that's a problem... He worked for other company and makes with that email address the commits, but no he works for us and uses other email address, but the GitHub login is still the same... @Yajo, do you still have the other email address registered here in GitHub?

**Comment by hbrunn** [comment]
I think the alias makes sense. @StefanRijnhart had changed commit addresses too, don't you run in the same problem?

**Comment by StefanRijnhart** [comment]
I did not mix addresses so I did not have this problem.

~BTW in the Netherlands it is not unusual that the employer owns the copyright, so in that case there needs to be a new CLA granted by this company.~ Not relevant I believe.

**Comment by hbrunn** [comment]
but why does it work when I pull stuff that `<redactedoldeviladdress>` committed?

**Comment by StefanRijnhart** [comment]
Seems like Github adds the login in the commit metadata https://github.com/gurneyalex/ak-oca-clabot/blob/master/clabot.py#L371

**Comment by hbrunn** [comment]
interesting, cool and a little bit scary wanting to look through the metadata in our source trees

**Comment by StefanRijnhart** [comment]
Pedro found an instance of this error with my old email though: https://github.com/OCA/account-financial-tools/pull/738#issuecomment-442967530

**Comment by yajo** [comment]
I do not own that old address anymore, but my CLA is OK. I think we can just ignore the CLA message.

**Review by lfreeke — COMMENTED** [review]
:+1:  (functional test)

File: partner_company_type/__manifest__.py
Comment (NL66278) [inline partner_company_type/__manifest__.py]: https

File: partner_company_type/__manifest__.py
Comment (NL66278) [inline partner_company_type/__manifest__.py]: No need for base, if other depends

File: partner_company_type/migrations/10.0.1.0.1/post-migration.py
Comment (NL66278) [inline partner_company_type/migrations/10.0.1.0.1/post-migration.py]: If we still reed the rename of the field, we do not need it here. You can use the old_name= keyword on the field definition.

File: partner_company_type/__init__.py
Comment (NL66278) [inline partner_company_type/__init__.py:2]: __init__.py files still by convention have the License line, only the copyright lines were dropped

**Review by NL66278 — CHANGES_REQUESTED** [review]
Small changes would be nice. See other comments. After that, good to go.

**Comment by NL66278** [comment]
@daramousk Hi George, are you going to take up the review comments on this one?

File: partner_company_type/__init__.py
Comment (NL66278) [inline partner_company_type/__init__.py]: maybe my comment was misleading. There are no coding lines in Odoo 11.0, because it is python 3.x

**Review by NL66278 — APPROVED** [review]
:+1:  One additional comment todo. And needs rebase to squash Transifex en fixup! comits. For the rest: LGTM and tested. So in principle approve.

**Comment by NL66278** [comment]
@daramousk And it helps if you remove the WIP.

**Comment by daramousk** [comment]
@lfreeke Need your approval here as well

**Review by lfreeke — APPROVED** [review]
(functional test) :+1:

**Comment by Nikul-Chaudhary** [comment]
@daramousk Travis :red_circle:

**Comment by NL66278** [comment]
Travis error is in base_location_nuts. Nothing to do with this PR.

**Comment by NL66278** [comment]
Another example of why unit tests should not depend on the availability of external services...

**Comment by pedrobaeza** [comment]
@NL66278 if you merge a migration PR, please remember to check the mark in the migration issue for marking the module migration as completed. I have already done it for the modules you have merged today.

**Comment by NL66278** [comment]
@pedrobaeza Thanks. I will try to remember it.

**Pull request merged**
